opentelemetry-jaeger/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+## v0.21.0
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/pull/1487 

There was a bad commit and they already added v0.21.0, we'll need to rebase this to fix it.

Reply:
lets also add a note that : This will be the last release of this crate.

Reply:
along with the note of last relase I'd also note the propagator will move to `opentelemetry-jaeger-propagator`

Reply:
resolving this, as this is taken care by https://github.com/open-telemetry/opentelemetry-rust/pull/1560